pkg/file/xar.go

```diff
@@ -270,15 +276,22 @@ func parseDistributionFile(rawXML []byte) (*InstallerMetadata, error) {
 		BundleIdentifier: identifier,
 		PackageIDs:       packageIDs,
 	}, nil
+}
 
+// Set of package names we know are incorrect. If we see these in the Distribution file we should
+// try to get the name some other way.
+var knownBadNames = map[string]struct{}{
+	"DISTRIBUTION_TITLE": {},
+	"MacFULL":            {},
```

Comment:
`MacFULL` and `SU_TITLE` are two other "bad names" that @jmwatts found, as we find more we can add them to this set.